src/deck-options.md

```diff
@@ -2,262 +2,223 @@
 
 <!-- toc -->
 
-Deck options primarily control the way Anki schedules cards. It is recommended
+Deck options primarily control the way Anki schedules cards. If you're a new user, it's recommended
 that you spend a few weeks with the defaults to get a feel for how Anki works
-before you start adjusting options. Please make sure you understand the options
+before you start adjusting the options. Please make sure you understand the options
 before changing them, as mistakes could reduce Anki's effectiveness.
 
-Deck options are accessed by:
+If you're on the desktop app, do any of the following to access deck options:
 
-- Clicking the gear icon on the `Decks` screen.
-- Selecting a deck on the `Decks` screen, and then clicking `Options`
+- Click the gear icon on the Decks screen.
+- Select a deck on the Decks screen, and then click "Options"
   at the bottom of the screen.
-- Clicking on `More` > `Options` while in review mode.
-- Pressing `o` while in review mode.
+- Click "More" > "Options" while in review mode.
+- Press <kbd>o</kbd> while in review mode.
 
-This page describes the options shown in Anki 2.1.45+, when you have the v2 or
-v3 scheduler enabled. On older versions, some options will not be available, or
-will appear in a different section. Please keep in mind that the v1 scheduler
-is no longer supported in Anki 2.1.50+. If you have not yet updated to v2 or v3,
-you will be prompted to update when you attempt to review cards in 2.1.50+.
-
-For more info on deck options, please check:
+Here are some community posts about deck options:
```

Comment:
```suggestion
This page describes the options shown in Anki 2.1.45+, with the
v3 scheduler. On older versions, some options will not be available, or
will appear in different sections. 

For some broad information about deck options, see these community posts:
```

Reply:
I don't understand the logic behind this. Documentation is written to be relevant for the times, and nobody gets confused when earlier versions have different or missing options. You've misjudged the reader if you think they will appreciate this factoid.

Reply:
You already proposed an overall guideline for the manual to stop documenting changes. I don't think that proposal has been generally accepted -- https://github.com/ankitects/anki-manual/issues/252#issuecomment-2300015450 -- in contrast with your proposal to reduce/corral references to v1/v2 -- https://github.com/ankitects/anki-manual/issues/233#issue-2458510129 . 

I edited this to be in line with the idea of those 2 proposals. (Although, both of them would benefit from some concrete information about what you're intending to change/move to implement them.)

Reply:
> Currently, various parts of the manual tells the user when a feature was added.

I think I found and have kept one of those here, but this one seemed a relic from the past so I removed.

Reply:
Also not resolved.